lib/timetrap/cli.rb

```diff
@@ -247,14 +248,18 @@ def edit
       end
 
       if Config['note_editor']
-        if args['-z']
+        if args['-c']
```

Comment:
🥦  If `-c` is provided it will clear even if a note is provided. In other words clearing is prioritized.